src/NodaTime/LocalDate.cs

```diff
@@ -581,6 +581,24 @@ int IComparable.CompareTo(object obj)
         public bool Equals(LocalDate other) => this == other;
 
         /// <summary>
+        /// Resolves this local date into a <see cref="ZonedDateTime"/> in the given time zone representing the
+        /// start of this date in the given zone.
+        /// </summary>
+        /// <remarks>
+        /// This is a convenience method for calling <see cref="DateTimeZone.AtStartOfDay(LocalDate)"/>.
+        /// </remarks>
+        /// <param name="zone">The time zone to map this local date into</param>
+        /// <exception cref="SkippedTimeException">The entire day was skipped due to a very large time zone transition.
+        /// (This is extremely rare.)</exception>
+        /// <returns>The <see cref="ZonedDateTime"/> representing the earliest time on this date, in the given time zone.</returns>
+        [Pure]
+        public ZonedDateTime InZone([NotNull] DateTimeZone zone)
```

Comment:
I'm not convinced that `InZone` is a great name for a `LocalDate` ↦ `ZonedDateTime` function that only returns a small portion of the latter domain.

How about `GetStartOfDay(DateTimeZone)`? (though I'm not too keen on "Get".)